src/table/utils/__tests__/get-cell-renderer.spec.js

```diff
@@ -5,8 +5,8 @@ import * as withStyling from '../../components/withStyling';
 
 describe('render', () => {
   describe('getCellRenderer', () => {
-    let selectionsEnabled: boolean;
-    let hasColumnStyling: boolean;
+    let selectionsEnabled;
+    let hasColumnStyling;
```

Comment:
why revert this file?

Reply:
Because it causes errors when doing yarn types:check

Reply:
yes, but could you actually fix the errors instead? Since we want to merge this asap, I guess we could come back to it. just make sure we don't lose track of it then

Reply:
hmm Is it the mockImplementation that is the problem?

Reply:
![Screenshot 2022-08-09 at 14 03 26](https://user-images.githubusercontent.com/4471489/183642522-22c41f4e-267e-4c20-bfad-2777ea82bc52.png)
Do you have a quick fix for the issue above?
I think we need to convert withSelections, withColumnStyling, and withStyling to typescript to fix it.
If no quick fix, I suggest that we look at it after merging this PR.

Reply:
yeah, that was my thought too. It is fine like this for now

Reply:
okay